### PR TITLE
[RW-1043][risk=no] code cleanup and increased test coverage for review columns

### PR DESCRIPTION
**Pull request by freemabd1**
code cleanup and increased test coverage for review columns

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java [hunk 1]
```diff
@@ -773,7 +772,7 @@ private ParticipantData convertRowToParticipantData(Map<String, Integer> rm,
         .valueNumber(bigQueryService.getString(row, rm.get("valueAsNumber")))
         .units(bigQueryService.getString(row, rm.get("units")))
         .itemDate(bigQueryService.getDateTime(row, rm.get("startDate")))
-        .domainType(DomainType.MASTER);
+        .domainType(DomainType.PHYSICAL_MEASURE);
```
Comment (osgsusan) [inline api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java]: Good catch

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java [hunk 2]
```diff
@@ -16,85 +16,84 @@
   private static final String TABLE_PREFIX = "p_";
   private static final String MASTER_TABLE = "p_all_events";
 
+  private static final String VISIT_COLUMNS =
+    ", visit_id as visitId\n" +
+      ", visit_concept_id as visitConceptId\n";
+
+  private static final String STANDARD_COLUMNS =
```
Comment (osgsusan) [inline api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Nice idea, cleaner overall

File: api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java (hunk 1 above)
Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java]: Thanks.

File: api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java [hunk 3]
```diff
@@ -841,35 +1110,19 @@ private void assertResponse(ParticipantDataListResponse response, PageRequest ex
...
-        assertThat(((Procedure) actualData).getStandardVocabulary()).isEqualTo(((Procedure) expected).getStandardVocabulary());
+        assertThat((Procedure) actualData).isEqualTo((Procedure) expected);
       } else if (expected instanceof Master) {
-        assertThat(((Master) actualData).getDataId()).isEqualTo(((Master) expected).getDataId());
-        assertThat(((Master) actualData).getSourceName()).isEqualTo(((Master) expected).getSourceName());
-        assertThat(((Master) actualData).getSourceValue()).isEqualTo(((Master) expected).getSourceValue());
-        assertThat(((Master) actualData).getSourceVocabulary()).isEqualTo(((Master) expected).getSourceVocabulary());
-        assertThat(((Master) actualData).getStandardName()).isEqualTo(((Master) expected).getStandardName());
-        assertThat(((Master) actualData).getStandardVocabulary()).isEqualTo(((Master) expected).getStandardVocabulary());
+        assertThat((Master) actualData).isEqualTo((Master) expected);
```
[the remark this answers is not in the document]
Comment (freemabd1) [reply api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java]: Done.

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java [hunk 4]
```diff
@@ -112,7 +111,7 @@
   public QueryJobConfiguration buildQuery(Long participantId,
                                           DomainType domain,
                                           PageRequest pageRequest) {
-    String tableName = DomainType.MASTER.equals(domain)
+    String tableName = DomainType.ALLEVENTS.equals(domain)
```
Comment (osgsusan) [inline api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Which just leaves MASTER_TABLE, but I'll leave that up to you.

Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Done.

**Pull request merged**